lib/Fhp/Segment/DME/ParameterTerminierteSEPASammellastschriftEinreichenV1.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Fhp\Segment\DME;
+
+use Fhp\Segment\BaseDeg;
+
+class ParameterTerminierteSEPASammellastschriftEinreichenV1 extends BaseDeg
+{
+    /** @var int Must be => 1 */
+    public $minimaleVorlaufzeitFNALRCUR;
```

Comment:
Das ist ein gutes Beispiel dafür, warum ich `Sepa` bevorzuge anstatt `SEPA`, denn dann stünde hier `FnalRcur`. Wenn du willst, kannst du es überall entsprechend anpassen.

Reply:
Finde es glaube besser wenn es SEPA bleibt. Das ist eine Feststehende Abkürzung. FNAL und RCUR sind Konstanten aus dem SEPA XML dazu. Würde ich auch groß lassen.

Reply:
Unser Style Guide sagt "Camel Case" und verlinkt auf [Wikipedia](https://en.wikipedia.org/wiki/Camel_case), wo der einleitende Satz vorschreibt "such that each word **or abbreviation** in the middle of the phrase begins with a capital letter ...", aber keine Vorschriften oder Beispiele zu den übrigen (nicht-ersten) Buchstaben bringt.

Die meisten Style-Guides, die dazu eine Meinung haben, sind für `Sepa` (bspw. [Microsoft](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions?redirectedfrom=MSDN)).
Aber viele Teile dieser Bibliothek verwenden schon all-caps, also können wir es hier durchaus so lassen. Ich wollte nur darauf hinweisen, dass dieser Variablenname (zwei aufeinanderfolgende Akronyme) ein gutes Beispiel dafür ist, warum die meisten Style Guides durchgehend Camel Case bevorzugen.